src/fheroes2/campaign/campaign_scenariodata.cpp

```diff
@@ -700,7 +700,7 @@ namespace Campaign
             return description;
         }
         case ScenarioBonusData::RESOURCES: {
-            std::string description( _amount > 0 ? _( "The kingdom will have by %{amount} more %{resource} at the start of the scenario." )
+            std::string description( _amount > 0 ? _( "The kingdom will receive by %{amount} more %{resource} at the start of the scenario." )
                                                  : _( "The kingdom will have by %{amount} less %{resource} at the start of the scenario." ) );
```

Comment:
Same as the above comment
```suggestion
                                                 : _( "The kingdom will have %{amount} less %{resource} at the start of the scenario." ) );
```